BitcoinCore/BitcoinCore/Core/BitcoinCore.swift

```diff
@@ -212,6 +217,19 @@ extension BitcoinCore {
         return dataProvider.debugInfo
     }
 
+    public var statusInfo: [(String, Any)] {
+        var status = [(String, Any)]()
+        status.append(("synced until", ((lastBlockInfo?.timestamp.map { Double($0) })?.map { Date(timeIntervalSince1970: $0) }) ?? "n/a"))
+
+        status.append(contentsOf:
+            peerManager.connected().enumerated().map { (index, peer) in
+                ("peer \(index + 1)", initialBlockDownload.isSynced(peer: peer) ? "synced" : "not synced")
```

Comment:
We need some more information about the peer. Like IP address, announcedBestBlockHeight, version